src/components/common/FolioNumber.vue

```diff
@@ -70,11 +70,16 @@ export default class FolioNumber extends Vue {
   @Prop({ default: null }) readonly initialValue!: string
   @Prop({ default: false }) readonly isEditing!: boolean
   @Prop({ default: false }) readonly showErrors!: boolean
+  @Prop({ default: false }) readonly folioNumberLabel!: boolean
```

Comment:
Added prop for passing label

Reply:
Another way to do this is for the prop to be the optional value of the new label. By default, the value would be the old value. What do you think?

Reply:
Another thing you could do is ask Yui if it's OK to simply update the label for all filings that use this common component.

Reply:
Yui confirmed both. Applying changes now

Reply:
Looks good. Resolving.